LibraryTests/LibraryTests.cs

```diff
@@ -0,0 +1,37 @@
+using BookShelfBL;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace LibraryTests
+{
+    [TestClass]
+    public class LibraryTests
```

Comment:
Please add unit tests for all public method in your services. And please do it after you will inject data to service with some IDataProvider etc. I want you used Moq framework for that.